packages/grafana-ui/src/components/Table/RowsList.tsx

```diff
@@ -350,7 +353,7 @@ export const RowsList = (props: RowsListProps) => {
       tableState.expanded,
       tableStyles,
       textWrapField,
-      theme.components.table.rowHoverBackground,
+      theme.colors.background.canvas,
```

Comment:
I'm not sure these current changes are doing anything from my testing locally 🤔 I found though that if we swap out the styles on this line with the `background.canvas` it does have an effect. Either way maybe we should split out the styling changes in a separate PR so @staton-hyse11 (our product designer) can review / not block the bug fix

Current main
<img width="955" alt="Screenshot 2024-07-11 at 8 18 07 PM" src="https://github.com/user-attachments/assets/6409ce24-2be9-4295-b586-26c5664fe12d">
<img width="952" alt="Screenshot 2024-07-11 at 8 14 30 PM" src="https://github.com/user-attachments/assets/1008c42a-fc70-4643-8570-397f9ce2f292">



Current PR
<img width="955" alt="Screenshot 2024-07-11 at 8 15 20 PM" src="https://github.com/user-attachments/assets/04b70ad5-d680-4a6e-997f-f8a5f5c1b43f">

<img width="957" alt="Screenshot 2024-07-11 at 8 14 39 PM" src="https://github.com/user-attachments/assets/d272e73e-92dd-4f2a-a662-e30fb876a280">




PR with modification specified
<img width="953" alt="Screenshot 2024-07-11 at 8 18 17 PM" src="https://github.com/user-attachments/assets/d9e9e47c-888c-4268-b911-453fbbb9a56d">
<img width="957" alt="Screenshot 2024-07-11 at 8 14 39 PM" src="https://github.com/user-attachments/assets/b888dac8-7d35-4316-8f38-e20aa9081a32">

Reply:
Yeah, great point, I kinda rushed the original PR adding this functionality and we were left with a background color so subtle that you can't see its selected on screenshares.

Made this PR to deal with that issue separately: https://github.com/grafana/grafana/pull/90364